docs/lake/design.md

```diff
@@ -362,7 +387,8 @@ The commits still appear in the log but scans at any time-travel point
 where the commit is present will fail to scan the deleted data.
 
 Alternatively, old data can be removed from the system using a safer
-command (but still in the DANGER-ZONE), `zed lake vacate`, which moves
+command (but still in the DANGER-ZONE), `zed lake vacate` (implementation
+tracked in [zed/2545](https://github.com/brimdata/zed/issues/2545)) which moves
```

Comment:
While I found the open issue for `zed lake vacate` (which is already shown in `zed lake -h` output) and linked to it, I see `zed lake purge` (mentioned a few lines above) is not yet in the `-h` output nor could I find an open issue. @mccanne: Is this still one we'd like to implement? I'm happy to open an additional issue, but figured I'd check first.